examples/usecases/ranking_with_multitask_learning.ipynb

```diff
@@ -0,0 +1,1663 @@
+{
```

Comment:
<p>typo: "there might <strong>be </strong>some event dependency in the system the user is interacting <strong>with</strong>"</p><p> </p><p>BTW this is a really cool feature!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/948/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>thanks @radekosmulski&nbsp;. We found these scenarios of some targets depending on other targets in our research experiments with AliCCP and TenRec. It seems a good practice, because without that the results for sparser targets are really poor</p>